### ISSUE TITLE
Support for bitcoincash: prefix with Scan and Sweep

### ISSUE THREAD

**Issue by cranau** [Bitcoin-com/Wallet#26]
If I scan a private key QR code, then the Bitcoin.com Wallet will offer the option to Sweep the address. If the address contains the `bitcoincash:` prefix, the Sweep option is _not_ offered.

**Comment by CreativeName4** [comment]
yes, please! it's the only thing holding us back from on baoarding a ton of new people with BCH paper wallets right now

**Comment by zquestz** [comment]
I would love for this to work. Have had a number of people contact me that sweeping cashaddress.org paper wallets doesn't work right in this wallet. Would love to get it resolved!

**Comment by Wecx** [comment]
A fundamental feature that should be available.

**Comment by checksum0** [comment]
This is duplicate of issue #11 from early september.

**Comment by cranau** [comment]
Scanning public keys works correctly in the current version of the iOS app. However, scanning private keys does not.

**Comment by rwcarlsen** [comment]
This is a significant barrier for me introducing many new people to bitcoin cash.  Please make this a priority.
